PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -22,6 +24,8 @@
     private Button startButton;
     private Button newUserButton;
     private Button aboutButton;
+    private int alert_icon = R.drawable.ic_launcher;
```

Comment:
@top-gun007 Since it's the logo of our app which is consistent ,you can go ahead with this method for unstyled drawable.